mlx/c/fast.cpp

```diff
@@ -8,6 +8,7 @@
 #include "mlx/c/mlx.h"
 #include "mlx/c/private/array.h"
 #include "mlx/c/private/closure.h"
+#include "mlx/c/private/io.h"
```

Comment:
Is this just for consistency with the other files?

Reply:
yes